src/thunderbots/software/ai/hl/stp/evaluation/robot.h

```diff
@@ -48,7 +46,11 @@ namespace Evaluation
      * @param timestamp The time at which we want to know if the robot had the ball
      * @return True if the ball is close to the front dribbler and false otherwise
      */
-    bool robotHasPossession(Ball ball, Robot robot, Timestamp timestamp = Timestamp());
+    bool robotHasPossession(const Ball& ball, const Robot& robot,
+                            Timestamp timestamp = Timestamp());
```

Comment:
Why are we giving this a default `timestamp`? Seems like this is something we would like to force the user to specify? Alternatively, we could make it a `std::optional<Timestamp>` and use the most recent one if the user gives `std::nullopt`? 

Similarly for `robotBeingPassedTo` below.

Reply:
This was to make refactoring less of a PITA during competition. will make the change